src/components/modeler/Modeler.vue

```diff
@@ -2113,7 +2113,7 @@ export default {
             this.promptSessionId = '';
             this.fetchHistory();
           } else {
-            window.ProcessMaker.alert(errorMsg, 'danger');
+            console.log(errorMsg, 'danger');
```

Comment:
better to use console.error as in WelcomeMessage.vue

Reply:
Fixed